src/VM/OpcodeHandler.h

```diff
@@ -27,6 +27,9 @@
 
 // Third party includes
 
+// Marks handler parameter as unused and prevents compiler warnings.
+#define UNUSED_ARG(arg) (void)(arg)
```

Comment:
It's also simpler to just remove the argument's name. Programmers understand it's unused and compilers don't complain, so it's probably a convention.